stat.h

```diff
@@ -237,6 +238,12 @@ struct thread_stat {
 	fio_fp64_t ss_deviation;
 	fio_fp64_t ss_criterion;
 
+	bool priority_bit;
+	uint64_t io_u_plat_prio[FIO_IO_U_PLAT_NR];
+	uint64_t io_u_plat_low_prio[FIO_IO_U_PLAT_NR];
+	struct io_stat clat_prio_stat __attribute__((aligned(8)));
+	struct io_stat clat_low_prio_stat __attribute__((aligned(8)));
+
```

Comment:
There's a lot of if `ddir == DDIR_READ...` in the sample printing code now. I wonder it wouldn't be better to just avoid the ifs and print zeros for those directions that priority wasn't gathered on.

Reply:
OK I've done my review for now and asked a few questions.

Reply:
We could print zeroes and add the other I/O types to the various priority specific structures, but this seemed like a cleaner implementation to me. But if you think it's better to print zeroes and allocate ddir arrays for the priority structures, then I'll be happy to change it.